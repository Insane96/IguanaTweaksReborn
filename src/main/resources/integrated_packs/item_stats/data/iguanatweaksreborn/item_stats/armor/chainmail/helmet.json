{
	"item": "minecraft:chainmail_helmet",
	"durability": 33,
	"armor": 1.8,
	"modifiers": [

	]
}