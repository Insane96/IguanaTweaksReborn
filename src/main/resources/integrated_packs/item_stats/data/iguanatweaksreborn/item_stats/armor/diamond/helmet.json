{
	"item": "minecraft:diamond_helmet",
	"durability": 195,
	"armor": 3,
	"modifiers": [
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Armor slowdown",
			"slot": "head",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.016,
			"operation": "multiply_base"
		}
	]
}