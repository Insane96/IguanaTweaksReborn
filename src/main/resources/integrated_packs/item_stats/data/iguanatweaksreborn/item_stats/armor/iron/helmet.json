{
	"item": "minecraft:iron_helmet",
	"durability": 104,
	"armor": 2,
	"knockback_resistance": 0.03,
	"modifiers": [
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Armor slowdown",
			"slot": "head",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.01,
			"operation": "multiply_base"
		}
	]
}