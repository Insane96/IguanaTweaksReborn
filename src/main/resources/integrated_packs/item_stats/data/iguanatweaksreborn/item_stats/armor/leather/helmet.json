{
	"item": "minecraft:leather_helmet",
	"durability": 65,
	"armor": 0.8,
	"modifiers": [
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Armor slowdown",
			"slot": "head",
			"attribute": "minecraft:generic.movement_speed",
			"amount": 0,
			"operation": "multiply_base"
		},
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Item stats data pack",
			"slot": "head",
			"attribute": "iguanatweaksreborn:innate_resistance",
			"amount": 0.4,
			"operation": "addition"
		}
	]
}