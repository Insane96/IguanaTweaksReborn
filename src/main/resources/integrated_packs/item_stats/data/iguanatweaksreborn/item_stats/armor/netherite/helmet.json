{
	"item": "minecraft:netherite_helmet",
	"durability": 182,
	"armor": 3,
	"modifiers": [
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Armor slowdown",
			"slot": "head",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.024,
			"operation": "multiply_base"
		}
	]
}