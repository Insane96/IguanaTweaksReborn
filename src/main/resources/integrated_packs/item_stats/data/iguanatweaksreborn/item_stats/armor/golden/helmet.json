{
	"item": "minecraft:golden_helmet",
	"durability": 65,
	"armor": 1,
	"armor_toughness": 0,
	"modifiers": [
		{
			"uuid": "3f22e9a3-0916-43ab-a93f-ba52e5ae28e5",
			"name": "Golden armor bonus max health",
			"slot": "head",
			"attribute": "minecraft:generic.max_health",
			"amount": 5,
			"operation": "addition"
		},
		{
			"uuid": "cf4db784-9677-4e90-9c74-ee3d85c9a641",
			"name": "Armor slowdown",
			"slot": "head",
			"attribute": "minecraft:generic.movement_speed",
			"amount": -0.004,
			"operation": "multiply_base"
		}
	]
}